Code/Tools/FBuild/Documentation/docs/functions/library.html

```diff
@@ -37,7 +37,7 @@ <h1>Library</h1>
   ; options for compilation
   .Compiler                 ; Compiler to use
   .CompilerOptions          ; Options for compiler
-  .CompilerOutputPath       ; Path to store intermediate objects
```

Comment:
CompilerOutputPath is conditionally optional. It's only optional in the special case of using ObjectList or Library to compile a precompiled header separately from the objects that consume it.

MetaOptional is used in the code and the emptiness is checked as a special case to manage this.

I think either way you look at it, this one-liner documentation can't communicate that special case effectively, so will always be wrong one way or another. I think it could be argued that listing it as required since that is the most common case might be preferable.

Reply:
Sounds good. I undid the change to `CompilerOutputPath`, both in `library.html` and in `objectlist.html`.